src/server/scripts/Spells/spell_evoker.cpp

```diff
@@ -203,6 +205,99 @@ class spell_evo_pyre : public SpellScript
     }
 };
 
+
+// Deep Breath 357210
+class spell_evo_deep_breath : public SpellScript // v1 without mechanic empowered spells gives errors
+{
+        bool Validate(SpellInfo const* /*spellInfo*/) override
+        {
+            return ValidateSpellInfo({ SPELL_EVOKER_DEEP_BREATH });
+        }
+      
+        void HandleHitTarget(SpellEffIndex /*effIndex*/)
+        {
+            Unit* caster = GetCaster();
+             Unit* hitUnit = GetHitUnit();
+        }
+
+        void HandleLaunchTarget(SpellEffIndex /*effIndex*/)
+        {
+            Unit* caster = GetCaster();
+
+            if (AuraEffect* auraEffect = caster->GetAuraEffect(SPELL_EVOKER_DEEP_BREATH, EFFECT_0))
+            {
+                int32 manaCost = GetSpell()->GetPowerTypeCostAmount(POWER_MANA).value_or(0);
+                if (manaCost != 0)
+                    GetCaster()->ModifyPower(POWER_MANA, CalculatePct(manaCost, auraEffect->GetAmount()));
+            }
+        }
+
+
+        void HandleDamage(SpellEffIndex /*effIndex*/)
+        {
+            GetCaster()->CastSpell(GetHitUnit()->GetPosition(), SPELL_EVOKER_DEEP_BREATH, true);
+
+        }
+
+        SpellCastResult CheckCast()
+        {
+            Unit* caster = GetCaster();
+
+            if (!caster->IsFalling())
+                return SPELL_FAILED_NOT_ON_GROUND;
+
+            return SPELL_CAST_OK;
+
+
+        }
+         //TODO: need id of SPELL_EVOKER_DEEP_BREATH_VISUAL,SPELL_EVOKER_DEEP_BREATH_EMPOWERED_BUFF  and empowered damage mechanic too
```

Comment:
https://www.wowhead.com/spell=353759/deep-breath#triggered-by (buff)
https://www.wowhead.com/spell=362010/deep-breath The Jump Charge.

Reply:
wao ok didnt know it had another id i will change it soon or add what missing